spec/services/events/record_spec.rb

```diff
@@ -602,4 +602,58 @@
       end
     end
   end
+
+  describe '.record_teacher_starts_mentoring_event!' do
+    let(:started_on_param) { { started_on: 2.years.ago.to_date } }
+    let(:school) { FactoryBot.create(:school) }
```

Comment:
Just FYI we've included `FactoryBot::Syntax::Methods` now, so you should be able to just do:

```suggestion
    let(:school) { create(:school) }
```

Reply:
We decided to omit `FactoryBot::Syntax::Methods` on purpose because otherwise it's not obvious where `#create` is defined or what it's doing. I think we should stick to being explicit.